Orange/widgets/gui.py

```diff
@@ -1195,6 +1195,36 @@ def attributeItem(var):
     return attributeIconDict[var], var.name
 
 
+class ListViewWithSizeHint(QListView):
+    def __init__(self, *args, preferred_size=None, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Given how one can pass a preferred size as a kwarg into QWidget constructor, e.g.
```py
QListView(parent, sizeHint=QSize(100, 80), **other_properties)
```
is this class necessary?

Yes, QSize needs to be imported.